docs/general/chain-state-values.md

```diff
@@ -250,6 +271,10 @@ A Kusama bounty has a predetermined duration of <RPC network="kusama" path="cons
 
 The waiting time before claiming a Kusama child bounty reward is <RPC network="kusama" path="consts.bounties.bountyDepositPayoutDelay" defaultValue={57600} filter="blocksToDays"/> days.
 
+#### Chill Threshold
```

Comment:
Unless I am missing something, it seems that some parameters are only Kusama or only Polkadot, when it seems a tabbed layout would be needed here to include parameters from both chains (assuming it's a general page?)